### PR TITLE
[documentation] Add information for the prefix that is added to every log record

### PR DESCRIPTION
**Pull request by violetagg**

### CODE REVIEW

**Review by pderop — COMMENTED** [review]
For the HTTP/2 part, it is described the following:

"_In case of `HTTP/2`, the following is added at the beginning of every log record: the id of the underlying connection,
**the number of the stream received on that connection**, local and remote addresses._"

But then, after, in the example it seems that the number of the stream received on that connection is indicated at the end of the log, not before the local addr:

Examples
[a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 - R:/[0:0:0:0:0:0:0:1]:53444](**H2 - 5**)
[a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 ! R:/[0:0:0:0:0:0:0:1]:53444](**H2 - 5**)

so, shouldn't the stream id (**H2 - 5**) be placed before the local addr in the example above ?

**Comment by violetagg** [comment]
> For the HTTP/2 part, it is described the following:
> 
> "_In case of `HTTP/2`, the following is added at the beginning of every log record: the id of the underlying connection, **the number of the stream received on that connection**, local and remote addresses._"
> 
> But then, after, in the example it seems that the number of the stream received on that connection is indicated at the end of the log, not before the local addr:
> 
> Examples [a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 - R:/[0:0:0:0:0:0:0:1]:53444](**H2 - 5**) [a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 ! R:/[0:0:0:0:0:0:0:1]:53444](**H2 - 5**)
> 
> so, shouldn't the stream id (**H2 - 5**) be placed before the local addr in the example above ?

Yeah it is at the end in the current implementation.

Should I change the description like that "the id of the underlying connection, local and remote addresses,
**the id of the stream received on that connection**."?

**Comment by violetagg** [comment]
> yes

please check now

File: docs/asciidoc/faq.adoc [hunk 1]
```diff
@@ -4,9 +4,84 @@
...
+Examples
+[a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 - R:/[0:0:0:0:0:0:0:1]:53444]
+[a1566d55, L:/[0:0:0:0:0:0:0:1]:53446 ! R:/[0:0:0:0:0:0:0:1]:53444]
+
+Format
+[<CONNECTION_ID>, L:<LOCAL_ADDRESS> <CONNECTION_OPENED_CLOSED> R:<REMOTE_ADDRESS>]
+<CONNECTION_ID>: a1566d55
+<LOCAL_ADDRESS>: [0:0:0:0:0:0:0:1]:53446
+<REMOTE_ADDRESS>: [0:0:0:0:0:0:0:1]:53444
+<CONNECTION_OPENED_CLOSED>: - (connection opened)
```
Comment (simonbasle) [inline docs/asciidoc/faq.adoc:34]: nitpick but I'd put that in the same order as it appears in the log line, ie. before REMOTE_ADDRESS line 33.
same for the other "format" sections (HTTP1, HTTP2...) below

Comment (violetagg) [reply docs/asciidoc/faq.adoc:33]: I fixed that, please check now

**Review by simonbasle — APPROVED** [review]

**Comment by violetagg** [comment]
@pderop @simonbasle Thanks for the review!

**Pull request merged**